neo3/wallet/account.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from neo3 import contracts, settings, wallet
+from neo3.core import cryptography, types
+
+
+class Account:
+
+    def __init__(self, script_hash: types.UInt160, nep2key: str = None):
+        self._script_hash: types.UInt160 = script_hash
+        self._nep2key: str = nep2key
+        self.label = ''
+        self.is_default = False
+        self.lock = False
+        self._key: cryptography.KeyPair = None
+        self.contract: contracts.Contract = None
+        self.extra = None
+
+    @property
+    def address(self) -> str:
+        return wallet.to_address(self._script_hash, settings.network.account_version)
+
+    @property
+    def decrypted(self) -> bool:
```

Comment:
would `unlocked` be a more self-explanatory name?

Reply:
NEP-6 has an attribute called `lock`, therefore, using `unlocked` as a name instead of `decrypted` would imply that this function returns `not self.lock`.